ext/test/http/curl_http_test.cc

```diff
@@ -329,7 +329,7 @@ TEST_F(BasicCurlHttpTests, SendGetRequestSyncTimeout)
   curl::HttpClientSync http_client;
 
   http_client::Headers m1 = {};
-  auto result             = http_client.Get("http://222.222.222.200:19000/get/", m1);
```

Comment:
Wondering why is the IP address changed? I think it is intended as some invalid/unreachable address. @lalitb

Reply:
I think this address is invalid either

Reply:
Some local app could listen on https://127.0.0.1:19000?

Reply:
We can also use the address from `Documentation Address Block` (https://datatracker.ietf.org/doc/html/rfc5737#section-4), as they are not allowed to be used on the public internet.
```
192.0.2.0 – 192.0.2.255
198.51.100.0 – 198.51.100.255
233.252.0.0 - 233.252.0.255
```